hardhat/test/fix-accounting.js

```diff
@@ -0,0 +1,78 @@
+const { ethers } = require("hardhat");
+const { expect } = require("chai");
+
+describe("CEtherDelegateTempExploitAccounting", function () {
+  it("Should merge the attacker's supply and borrow balances", async function () {
+    // Enable using 0 gas price
+    await hre.network.provider.send("hardhat_setNextBlockBaseFeePerGas", ["0x0"]);
+
+    // Deploy new CEtherDelegateTempExploitAccounting
+    const CEtherDelegateTempExploitAccounting = await ethers.getContractFactory("CEtherDelegateTempExploitAccounting");
+    var cEtherDelegateTempExploitAccounting = await CEtherDelegateTempExploitAccounting.deploy();
+
+    // Deploy new CEtherDelegate
+    const CEtherDelegate = await ethers.getContractFactory("CEtherDelegate");
+    var cEtherDelegate = await CEtherDelegate.deploy();
+
+    // Get pool 8 admin
+    const Comptroller = await ethers.getContractFactory("Comptroller");
+    var comptroller = Comptroller.attach("0xc54172e34046c1653d1920d40333dd358c7a1af4");
+    var comptrollerAdmin = await comptroller.admin();
+    
+    // Impersonate admin of pool 8
+    await hre.network.provider.request({
+        method: "hardhat_impersonateAccount",
+        params: [comptrollerAdmin],
+    });
+
+    // Get FuseFeeDistributor owner
+    var ffd = new ethers.Contract("0xa731585ab05fC9f83555cf9Bff8F58ee94e18F85", [{"inputs":[{"internalType":"address[]","name":"oldImplementations","type":"address[]"},{"internalType":"address[]","name":"newImplementations","type":"address[]"},{"internalType":"bool[]","name":"allowResign","type":"bool[]"},{"internalType":"bool[]","name":"statuses","type":"bool[]"}],"name":"_editCEtherDelegateWhitelist","outputs":[],"stateMutability":"nonpayable","type":"function"},{"inputs":[],"name":"owner","outputs":[{"internalType":"address","name":"","type":"address"}],"stateMutability":"view","type":"function"}], ethers.provider);
+    var ffdOwner = await ffd.owner();
+    
+    // Impersonate FuseFeeDistributor owner
+    await hre.network.provider.request({
+        method: "hardhat_impersonateAccount",
+        params: [ffdOwner],
+    });
+
+    // Call FuseFeeDistributor._editCEtherDelegateWhitelist
+    await ffd.connect(await ethers.getSigner(ffdOwner))._editCEtherDelegateWhitelist(["0xd77e28a1b9a9cfe1fc2eee70e391c05d25853cbf", cEtherDelegateTempExploitAccounting.address], [cEtherDelegateTempExploitAccounting.address, cEtherDelegate.address], [false, false], [true, true], { gasPrice: "0" });
+
+    // Get attacker's initial balances
+    var cEther = CEtherDelegate.attach("0xbB025D470162CC5eA24daF7d4566064EE7f5F111");
+    var exchangeRateStored = await cEther.exchangeRateStored();
```

Comment:
@sriyantra can we break all of these magic vars out into a config/library file? Will be easier to audit.